proteobench/modules/template/plot.py

```diff
@@ -6,25 +6,17 @@
 from streamlit_plotly_events import plotly_events
 
 
-def plot_bench1(result_df):
-    """Plot results with Plotly Express."""
-    # TODO create (plotly) figure object
-    fig = go.Figure()
+class PlotDataPoint:
+    def plot_bench1(result_df):
+        """Plot results with Plotly Express."""
+        # TODO create (plotly) figure object
+        fig = go.Figure()
 
-    return fig
+        return fig
 
+    def plot_bench2(result_df):
+        """Plot results with Plotly Express."""
+        # TODO create (plotly) figure object
+        fig = go.Figure()
 
-def plot_bench2(result_df):
-    """Plot results with Plotly Express."""
-    # TODO create (plotly) figure object
-    fig = go.Figure()
-
-    return fig
-
-# ? Should we define two function to implement instead of an ABC?
```

Comment:
So you have chosen to not use an ABC, but a class after all? The class has no state and the module already groups the two functions. I would just use plain functions here.

Reply:
I would just provide empty functions raising `NotImplementedError`s for the template. I'll revert the changes and make a commit.